0028-web-monetization/0028-web-monetization.md

```diff
@@ -1,79 +1,296 @@
 ---
 title: Web Monetization
```

Comment:
Why is this called Web Monetization instead of something like the Web Micropayment API?